src/components/app-configure/store/UserLimits.vue

```diff
@@ -0,0 +1,234 @@
+<script lang="ts" setup>
+import { computed, type ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VDropdown from '@/components/lib/VDropdown/VDropdown.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import VTextField from '@/components/lib/VTextField/VTextField.vue'
+import type {
+  UserLimitParam,
+  UserLimitTarget,
+  UserLimitUnit,
+} from '@/store/configure.store'
+import {
+  userLimitOptions,
+  storageValues,
+  bandwidthUnits,
+  type BandwidthLimitUnit,
+} from '@/utils/constants'
+
+type UserLimitKind = 'Unlimited' | 'Limited'
+type UserLimit = {
+  isUnlimited: boolean
+  value?: number
+  unit?: UserLimitUnit
+}
+
+const store = useStore()
+
+const storageLimitKind: ComputedRef<UserLimitKind> = computed(() => {
+  if (store.getters.storageLimit.isUnlimited) {
+    return 'Unlimited'
+  }
+  return 'Limited'
+})
+
+const bandwidthLimitKind: ComputedRef<UserLimitKind> = computed(() => {
+  if (store.getters.bandwidthLimit.isUnlimited) {
+    return 'Unlimited'
+  }
+  return 'Limited'
+})
+
+const storageLimit: ComputedRef<UserLimit> = computed(
+  () => store.getters.storageLimit
+)
+const bandwidthLimit: ComputedRef<UserLimit> = computed(
+  () => store.getters.bandwidthLimit
+)
+
+const bandwidthLimitUnit: ComputedRef<BandwidthLimitUnit | undefined> =
+  computed(() => {
+    if (!store.getters.bandwidthLimit.isUnlimited) {
+      return bandwidthUnits.find(
+        (bandwidthUnit) =>
+          bandwidthUnit.value === store.getters.bandwidthLimit.unit
+      )
+    } else {
+      return undefined
+    }
+  })
+
+function handleIsUnlimitedChange(type: UserLimitTarget, value: UserLimitKind) {
+  const isUnlimited: boolean = value === 'Unlimited' ? true : false
+  return store.commit('updateUserLimit', { type, isUnlimited })
+}
+
+function handleUserLimitFieldChange({ type, field, value }: UserLimitParam) {
+  return store.commit('updateUserLimitField', { type, field, value })
+}
+</script>
+
+<template>
+  <section name="user-limits">
+    <SettingCard>
+      <template #title>User Limits</template>
+      <template #description
+        >If you would like to limit the bandwidth or storage for each of your
+        users, please specify the same below. <a>LEARN MORE</a></template
+      >
```

Comment:
looks like formatting issue, can you check @shrinathprabhu

Reply:
This is controlled by the `htmlWhitespaceSensitivity` config in Prettier. The rule helps avoid leading and trailing spaces in inline elements which get rendered on the page. See this [description](https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting).